waltz-tools/src/main/java/com/wepay/waltz/tools/cluster/ClusterCli.java

```diff
@@ -289,6 +282,36 @@ protected void processCmd(CommandLine cmd) throws SubCommandFailedException {
             }
         }
 
+        private void updateRemainingValidationResults(CompletableFuture<Void> consistencyValidationFuture,
```

Comment:
We don't have to do it but it's just an idea. We could have the method take a Map<Future,ValidationType>> to have it more generic. So can then loop instead of type the same code twice below.

Reply:
makes sense. 👍